src/aura/GE_GiftEntryForm/GE_GiftEntryFormController.js

```diff
@@ -1,4 +1,33 @@
 ({
+    doInit: function(cmp) {
+        var action = cmp.get("c.createDefaultTemplate");
+
+        action.setCallback(this, function(response) {
+            var state = response.getState();
+
+            if (state === "SUCCESS") {
+
+                cmp.set('v.finishedInit', true);
+            }
+            else if (state === "INCOMPLETE") {
+                // do something
+            }
+            else if (state === "ERROR") {
+                var errors = response.getError();
+                
+                if (errors) {
+                    if (errors[0] && errors[0].message) {
+                        console.log("Error message: " + 
```

Comment:
We may need to think of a way to display this error, and we should remove the console.log